sql_metadata/utils.py

```diff
@@ -13,5 +13,22 @@ def append(self, item: Any) -> None:
         if item not in self:
             super().append(item)
 
+    def extend(self, items: Any) -> None:
```

Comment:
Let's make it explicit that `items` is expected to be [the sequence](https://docs.python.org/3/library/stdtypes.html#typesseq).

```suggestion
    def extend(self, items: Sequence[Any]) -> None:
```